### ISSUE TITLE
Is there a way to match an operator in a Vec ?

### ISSUE THREAD

**Issue by Teln0** [lalrpop/lalrpop#536]
I'm passing as a parameter a Vec of Strings representing the list of possible operators. Is it possible to have something like

```rust
pub Operator: String = {
    <o:*something that checks if the string is in the vector*> => o
}
```

PS : Sorry if this is the wrong place to ask questions.

**Comment by Marwes** [comment]
No, you should implement a custom lexer if you need that. Then you can pass the vector to the lexer and it can use the vector to interpret whether something is an operator.
